lib/infra_conversion_throttler.rb

```diff
@@ -80,6 +80,11 @@ def self.running_conversion_jobs
   # Applying the limits is done via the conversion_host which handles the writing.
   def self.apply_limits
     running_conversion_jobs.each do |ch, jobs|
+      if ch.nil?
+        _log.error("Migration task for '#{migration_task.source.name}' is running but has no conversion host")
```

Comment:
`migration_task.source.name` <- is the issue here, there's no `migration_task` yet here.

If you want to list the names of all of them you could do something like `source_names = jobs.map { |job| job.migration_task&.source&.name }.compact.join(", ")` and if you expect this could be a lot of vms you can limit it with `.compact.take(max_number_to_print)`